src/SDKs/Batch/DataPlane/Azure.Batch/CloudJob.cs

```diff
@@ -573,7 +573,39 @@ public void Refresh(DetailLevel detailLevel = null, IEnumerable<BatchClientBehav
             asyncTask.WaitAndUnaggregateException(this.CustomBehaviors, additionalBehaviors);
         }
 
-#endregion // CloudJob
+        /// <summary>
+        /// Gets the task counts for the job.
+        /// </summary>
+        /// <param name="detailLevel">A <see cref="DetailLevel"/> used for controlling which properties are retrieved from the service.</param>
```

Comment:
Hah, actually a great catch, because this API _doesn't_ support any OData queries.

I'm going to remove this from this API for now since it'll lure people into doing something that won't work.